notebooks/using-mlflow/train-with-mlflow/xgboost_nested_runs.ipynb

```diff
@@ -0,0 +1,955 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "trials, best_params = search(space, X_train, X_test, y_train, y_test)"
```

Comment:
Error:
![image](https://user-images.githubusercontent.com/9068012/171488104-300da243-c9c3-47c7-88e2-268f6ff9d81f.png)

Reply:
Environment installation issue in CI.